src/main/java/com/amazon/opendistroforelasticsearch/jdbc/protocol/http/JsonCursorQueryRequest.java

```diff
@@ -0,0 +1,39 @@
+package com.amazon.opendistroforelasticsearch.jdbc.protocol.http;
+
+import com.amazon.opendistroforelasticsearch.jdbc.protocol.Parameter;
+import com.amazon.opendistroforelasticsearch.jdbc.protocol.QueryRequest;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class JsonCursorQueryRequest implements QueryRequest  {
```

Comment:
Could you add some JavaDoc for new classes?

Reply:
Yes, working on it.

Reply:
+1, please helping add test coverage report and verification also. Let's set the lower bar as existing code code coverage and keep improving it. sample configuration
```
jacoco {
    toolVersion = "0.8.3"
}
jacocoTestReport {
    reports {
        html.enabled true
    }
}
test.finalizedBy(project.tasks.jacocoTestReport)
jacocoTestCoverageVerification {
    violationRules {
        rule {
            limit {
                minimum = 0.8
            }
        }
    }
}
check.dependsOn jacocoTestCoverageVerification
```

Reply:
Added jacoco for test coverage and enforcement, but looks its not being enforced. Checked by increasing values till `minimum = 1`.

Add unit tests and javadocs where possible.

Reply:
Thanks!